product-versions.json

```diff
@@ -0,0 +1,240 @@
+{
+  "schemaVersion": "1.0",
+  "channels": [
+    {
+      "name": "stable",
+      "products": [
+        {
+          "name": "Azure IoT Edge",
+          "id": "aziot-edge",
+          "version": "1.3.0",
+          "componentTypes" : [
+            {
+              "name": "dockerImage",
+              "platforms": [
+                {
+                  "os": "linux",
+                  "arch": [
+                    "amd64",
+                    "arm64v8",
+                    "arm32v7"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "package",
+              "platforms": [
+                {
+                  "os": "debian",
+                  "version": "11",
+                  "arch": [
+                    "armhf"
```

Comment:
How do we anticipate this info on arch being used?  If it were to be part of an `iotedge check` to see if there is a newer version then we'd probably need to include x86_64 and arm64 since those packages are "shipped" via our github release page.  The current list aligns with what we officially test and publish to packages.microsoft.com.

Reply:
I added the edgelet package information in commit 9e44b02 mostly to get an idea of what it might look like. But since I'm focused on Docker images right now, I don't think we really know how it will be used yet. You raise good points about whether `iotedge check` would use it, and whether we list packages for Tier 1 platforms only (PMC), or include packages uploaded to our GitHub release pages.

I think it is best to remove the "package" component type and related information for the first iteration of this JSON file. We can add it later, when we're ready to think through everything we need.

Reply:
> I think it is best to remove the "package" component type and related information for the first iteration of this JSON file. We can add it later, when we're ready to think through everything we need.

ok with me

Reply:
Perhaps another future improvement/addition to consider could be incorporating the SPDX json output of the [SBOM tool](https://github.com/microsoft/sbom-tool)

Reply:
Removed the "package" component type. We can add it back later when we have concrete usage scenarios.